### PR TITLE
feat: add beta distribution

### PR DESCRIPTION
**Pull request by WenheLI**

### CODE REVIEW

**Comment by WenheLI** [comment]
We should consider overflow & underflow for extreme value.

**Comment by yorkie** [comment]
How about adding some unit tests for this function?

**Comment by WenheLI** [comment]
@yorkie - Testing for random functions is kinda tricky. I am trying to find a way of testing.

File: test/node/rand/index.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { beta } from '../../../src/rand';
+import { expect } from 'chai';
+import * as tf from '@tensorflow/tfjs-core';
+import '@tensorflow/tfjs-backend-cpu';
+import 'mocha';
+
+describe('Beta Random test',
+  () => { 
+  it('should return tensor wiht one value', () => {
```
Comment (risenW) [inline test/node/rand/index.ts]: A little typo here:` it('should return tensor wiht one value')...` should be `it('should return tensor with one value'`

File: test/node/rand/index.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    const result = beta(1, 1);
+
+    expect(result.shape).to.eql([1]);
+    expect(result.dtype).to.eql('float32');
+    expect(result.dataSync()).to.eql(new Float32Array(1).fill(1));
+  });
+
+  it('should return tensor wiht multi value', () => {
```
Comment (risenW) [inline test/node/rand/index.ts]: Same typo mistake here

**Review by risenW — CHANGES_REQUESTED** [review]

**Comment by risenW** [comment]
> @yorkie - Testing for random functions is kinda tricky. I am trying to find a way of testing.

True, except you find a way to seed the function

File: package.json [hunk 3]
```diff
@@ -14,9 +14,10 @@
     "test": "npm run test:node && npm run test:browser",
     "test:node": "nyc mocha --timeout 10000 -r ts-node/register ./test/node/**/*.ts",
     "test:browser": "npm run build:browser && karma start --single-run --browsers ChromeHeadless karma.conf.js",
-    "build": "npm run clean && npm run build:node && npm run build:browser",
+    "build": "npm run clean && npm run build:node && npm run build:browser && npm run compile",
     "build:node": "tsc -b tsconfig.json",
     "build:browser": "npm run build:node && webpack --mode production",
+    "compile": "make",
```
Comment (yorkie) [inline package.json:20]:
```suggestion
    "build:wasm": "make",
```

File: src/rand/beta.ts [hunk 4]
```diff
@@ -0,0 +1,7 @@
+export declare class Beta {
+  constructor(a?: number, b?:number)
+
+  generate(): number;
+
+  setParam(a: number, b: number): void;
```
Comment (yorkie) [inline src/rand/beta.ts:6]: Add some description?

**Comment by WenheLI** [comment]
@yorkie gentle ping

**Review by yorkie — APPROVED** [review]

**Comment by WenheLI** [comment]
@risenW - Gentle ping

**Review by risenW — APPROVED** [review]

**Comment by risenW** [comment]
@WenheLI looks good

**Pull request merged**